packages/engine/src/scene/functions/loaders/BoxColliderFunctions.ts

```diff
@@ -54,6 +54,19 @@ export const deserializeBoxCollider: ComponentDeserializeFunction = (
   obj3d.traverse((mesh: Mesh) => {
     if (typeof mesh.userData['type'] === 'string') meshObjs.push(mesh)
   })
+
+  obj3d.userData.updateTransform = function () {
```

Comment:
this should be moved to updateBoxCollider, any changes to the object should cause that function to run

Reply:
No need to add this function to `updateBoxCollider` but it would be better to create an independent function which accepts `Entity` or `Object3d` and does this operation.

Reply:
yes good idea